scst-store/troubleshooting.hbs.md

```diff
@@ -83,7 +83,45 @@ SCST - Store uses the default storage class which uses EBS volumes by default on
 
 Follow the AWS documentation to install the Amazon EBS CSI Driver before installing SCST - Store or before upgrading to Kubernetes v1.23. For more information, see the [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html).
 
-## <a id="cert-expiries"></a> Certificate Expiries
+## <a id="ca-cert-expires"></a> CA Cert expires
+
+### <a id="ca-cert-expires-symptom"></a>Symptom
+
+The Insight CLI or Scan Controller fails to communicate with the SCST - Store and receives an error message containing the following text:
+
+```console
+tls: failed to verify certificate: x509: certificate has expired or is not yet valid
+```
+
+### <a id="ca-cert-expires-explanation"></a>Explanation
+
+The CA certificate expired before the app certificate expires, which causes the error even though the app certificate is still valid. 
+Certmanager will rotate the expired CA certificate, but it doesn't rotate the certificates that were previously created by the expired CA certificate. So this leads to contour being in a bad state and the certificates won't be refreshed properly.
+
+### <a id="ca-cert-expires-solution"></a>Solution
+
+1. Delete the existing expired cacert
+```console
+kubectl delete secret cacert contourcert envoycert -n projectcontour
+```
+2. Delete the contour-certgen job
+```console
+kubectl delete job contour-certgen -n projectcontour
+```
+3. Apply a new certgen
+```console
+kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour/main/examples/contour/02-job-certgen.yaml
```

Comment:
this may not be the best thing to do, in tap. We need the kapp-controller reconciler to force a reapply on the certgen. The reason this worked in our acceptance environment was because we manually installed contour not via tap

Reply:
So maybe something like `kctrl package installed kick --package-install contour -n tap-install` (or whatever the package-install name in TAP is called. I don't know what the package-install is actually called in tap 😅 )

Reply:
I checked with our acceptance cluster and looks like that name is right! :D